tests/functionnal/test_book.py

```diff
@@ -708,7 +708,7 @@ def test_book_add_content(self, test_client, headers):
 
         response = test_client.post(
             "/api/book/", headers=headers, json=dict(
```

Comment:
```suggestion
            "/api/book", headers=headers, json=dict(
```
Ne jamais mettre de "/" à la fin d'une url si tu ne mets rien derrière